ui/admin/app/components/form/group/add-members/index.hbs

```diff
@@ -1,3 +1,4 @@
+{{!-- template-lint-disable no-whitespace-for-layout  --}}
```

Comment:
Had to add turn off the individual rule here and in `add-principals` as it seems we're using layout whitespace as part of our design and have a TODO to come back and fix this.

Reply:
Feel free to create a Jira task under tech debt 😉

Reply:
Good point, I'll create tickets for these!